docs/error-api.asciidoc

```diff
@@ -1,11 +1,25 @@
+[[error-api]]
 == Error API
 
+The APM Server exposes an API Endpoint to send error records. 
+Unless you are implementing an agent, you don't need to know about the specifics of this API.
+
 To send error records you need to send a HTTP POST request to APM Server `errors` endpoint. Information pertaining to the error record must be sent as a JSON object to the endpoint.
 
 --
-See <<error-api-examples,Examples>> for sending a request to the error endpoint.
 
-See the schema definitions for how the error payload has to look like:
+Find more information about: 
+
+* <<error-schema-definition>>
```

Comment:
If there isn't a text after the link it will be rendered in cursive (`<<foo>>` vs `<<foo,Foo>>`).

So I personally always include the text even if it's the same as the headline that the link is pointing to. This way we'll not get mixed text-styling in case another bullet is added later where the text is included.

But maybe I'm doing it wrong 😅

Reply:
I don't see it being rendered in cursive.

If you have a look at the [current doc](https://www.elastic.co/guide/en/apm/server/current/_transaction_api.html) and the [plaing asciidoc](https://raw.githubusercontent.com/elastic/apm-server/master/docs/error-api.asciidoc) it is also linked without a name and not rendered cursive. Thus I think this should be fine.

Reply:
Weird - It must be yet another one of those differences with how it renders locally vs online. Stick to you what you got then 👍

Reply:
I actually prefer not specifying link text and letting the build resolve the text. This is especially important if you are referring to a full topic heading. It gets weird for users if you change the topic heading, but don't remember to change the link text. It's a maintenance issue. If you do use link text, it should be generic enough that users won't be confused if the actual topic heading changes.

Reply:
@dedemorton I agree 👍 For some reason when I rendered the Node docs locally, these kind of links looked different than links with a link text. But apparently that's just because our local build script doesn't match the one we use in production.

Reply:
@watson It's possible that Clint does some extra magic, but doesn't your script call https://github.com/elastic/docs/blob/master/build_docs.pl? AFAIK, the doc build simply calls build_docs.pl with --all specified (and uses the conf.yaml to set parameters).

Reply:
@dedemorton Yes, I'm using the build_docs.pl script. But for some reason this is what I'm seeing:

In the Node.js docs, if I write:

```asciidoc
* <<express,Get started with Express>>
* <<hapi>>
* <<koa,Get started with Koa>>
```

It's generated like so locally:

<img width="334" alt="screen shot 2017-09-20 at 10 13 48" src="https://user-images.githubusercontent.com/10602/30633575-6fc63f8a-9dec-11e7-9a9d-7e5d66a4db70.png">